docs/GreenFieldExample.md

```diff
@@ -0,0 +1,96 @@
+# GreenField API example with cURL
+
+The [GreenField API](./API/Greenfield/v1/) (also available on your instance on `/docs`) allows you to operate BTCPay Server via an easy to use REST API.
+
+Note that you can partially generate clients in the language of your choice by using the [Swagger file](./API/Greenfield/v1/swagger.json).
```

Comment:
Broken link, tests caught it.

Reply:
Those should be absolute links, becausee Vuepress doesn't know about these files.